src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.shared.cs

```diff
@@ -227,6 +262,46 @@ public void UpdateVolume()
 	/// Invokes the platform functionality to update the media playback volume.
 	/// </summary>
 	protected virtual partial void PlatformUpdateVolume();
+
+	/// <summary>
+	/// Sets the Navbar and Status Bar.
+	/// </summary>
+	/// <param name="fullScreenStatus"></param>
+	protected void SetBarStatus(bool fullScreenStatus)
+	{
+#if IOS || MACCATALYST
+		UIKit.UIApplication.SharedApplication.SetStatusBarHidden(fullScreenStatus, UIKit.UIStatusBarAnimation.Fade);
```

Comment:
`SetStatusBarHidden` is obsolete.

We should use the recommended API, `SetStatusBarHidden`, unless there's a technical limitation. 

<img width="1609" alt="image" src="https://user-images.githubusercontent.com/13558917/236954301-2beab8cd-e68c-456a-8960-a0030e62e46b.png">

Reply:
I will work on implementing this. It might take a few days for me to figure out. I will do my best.

Reply:
I tried everything I could think of. No luck. It is most likely user error on my part. If anyone else wants to try and figure this out I would love any sort of help.  I Have spent about half a day trying to solve just this one problem so far.

Reply:
Does anyone have any suggestions on how to implement ```PrefersStatusBarHidden()``` ? I have found it is not working and/or not yet implemented and it is listed in platform specific code bug report as not working yet.